CHANGES.txt

```diff
@@ -15,6 +15,7 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
       SConstruct/SConscript will still cause "scons: Reading SConscript files ..." to be
       printed, since the option is not set when the build scripts first get read.
     - Added check for SONAME in environment to setup symlinks correctly (Github Issue #3246)
+    - fix sser callable's expanded commands silently fail TypeError exceptions (Github Issue #3654)
```

Comment:
typo.

Reply:
Can you flesh this out a little?
 What causes this issue. Ideally our CHANGES.txt makes it possible for someone who has an issue to find out if a version fixes it.

Reply:
I'm guessing **sser** was supposed to be **user**?

Reply:
updated and expanded